hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkMergeHelper.java

```diff
@@ -101,13 +101,13 @@ public void runMerge(HoodieTable<T, JavaRDD<HoodieRecord<T>>, JavaRDD<HoodieKey>
     } catch (Exception e) {
       throw new HoodieException(e);
     } finally {
+      if (null != wrapper) {
```

Comment:
Can you please rename `wrapper` to `executor` as well? `wrapper` is a complete misnomer

Reply:
Done

Reply:
> Done
Revert it and may do this in another PR...